Traceback (most recent call last):
  File "/g/data/e14/as3189/OFAM/scripts/create_file_wsc.py", line 77, in <module>
    du = xr.open_mfdataset(u, combine='by_coords')
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/xarray/backends/api.py", line 959, in open_mfdataset
    datasets, compat=compat, data_vars=data_vars, coords=coords, join=join
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/xarray/core/combine.py", line 719, in combine_by_coords
    join=join,
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/xarray/core/merge.py", line 793, in merge
    merge_result = merge_core(dict_like_objects, compat, join, fill_value=fill_value)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/xarray/core/merge.py", line 555, in merge_core
    variables, out_indexes = merge_collected(collected, prioritized, compat=compat)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/xarray/core/merge.py", line 228, in merge_collected
    merged_vars[name] = unique_variable(name, variables, compat)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/xarray/core/merge.py", line 137, in unique_variable
    equals = getattr(out, compat)(var)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/xarray/core/variable.py", line 1722, in no_conflicts
    return self.broadcast_equals(other, equiv=equiv)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/xarray/core/variable.py", line 1703, in broadcast_equals
    return self.equals(other, equiv=equiv)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/xarray/core/variable.py", line 1687, in equals
    self._data is other._data or equiv(self.data, other.data)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/xarray/core/duck_array_ops.py", line 234, in array_notnull_equiv
    flag_array = (arr1 == arr2) | isnull(arr1) | isnull(arr2)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/dask/array/core.py", line 1838, in __eq__
    return elemwise(operator.eq, self, other)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/dask/array/core.py", line 3894, in elemwise
    name = kwargs.get("name", None) or "%s-%s" % (funcname(op), tokenize(op, dt, *args))
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/dask/base.py", line 652, in tokenize
    return md5(str(tuple(map(normalize_token, args))).encode()).hexdigest()
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/dask/utils.py", line 506, in __call__
    return meth(arg, *args, **kwargs)
  File "/g/data3/hh5/public/apps/miniconda3/envs/analysis3-20.01/lib/python3.7/site-packages/dask/base.py", line 874, in normalize_array
    data = hash_buffer_hex(x.ravel(order="K").view("i1"))
MemoryError: Unable to allocate 2.36 TiB for an array with shape (323835494400,) and data type float64
